examples/zolertia/zoul/Makefile

```diff
@@ -3,12 +3,12 @@ DEFINES+=PROJECT_CONF_H=\"project-conf.h\"
 CONTIKI_PROJECT = zoul-demo test-tsl2563 test-sht25 test-pwm test-power-mgmt
 CONTIKI_PROJECT += test-bmp085-bmp180 test-motion test-rotation-sensor
 CONTIKI_PROJECT += test-grove-light-sensor test-grove-loudness-sensor
-CONTIKI_PROJECT += test-weather-meter test-grove-gyro test-lcd
+CONTIKI_PROJECT += test-weather-meter test-grove-gyro test-lcd test-voc
```

Comment:
I missed this one: `test-voc` is a leftover from a previous change, rename to `test-iaq`